docs/STABLE.md

```diff
@@ -1,18 +1,18 @@
 Moving the stable tag
 =====================
 
-Because we do not have a PyPI package where folks can just install latest, our `README.md` instructions describe installing from the GitHub using a "stable" tag.
+Because we do not have a PyPI package where folks can just install latest, our `README.md` instructions describe installing from the GitHub using a `stable` tag.
 
 To support this, we move the stable tag to the latest release, as needed.
 
-This can be done by:
+This can be done via the following steps:
 
 Get current:
 
 ```ShellSession
 $ git switch main
-$ git fetch
-$ git pull
+$ git fetch upstream
```

Comment:
The use of `upstream` is sort of developer dependent. That is, the convention of naming the remote for the main repo to `upstream` is not universal. I call it `origin`. So you should probably state that the remote name `upstream` used in this doc refers to the main repo.

An example below still pushes to origin :-)

Reply:
Fair enough! I'll go back to what was there :+1: